Command/DumpCommand.php

```diff
@@ -81,6 +87,20 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this
             ->getContainer()
             ->get('bazinga.jstranslation.translation_dumper')
-            ->dump($this->targetPath, $formats, $merge);
+            ->dump($this->targetPath, $this->getTranslationsPath($input->getOption('path')), $formats, $merge);
+    }
+
+    /**
+     * Get translations path from the router defined or use the default one.
+     */
+    private function getTranslationsPath($path)
+    {
+        $translationsPathOverriden = empty($path) ? 'translations' : $path;
+        $route = $this->getContainer()->get('router')->getRouteCollection()->get('bazinga_jstranslation_js');
```

Comment:
so the CLI option only receives a pattern like ```/translations/{domain}.{_format}```.

TranslationDumper receives this pattern if the CLI is used. If it does not receive it (meaning that the CLI was not used), it should use the router pattern instead. Right?